guides/common/modules/proc_cloning_satellite_server.adoc

```diff
@@ -162,6 +162,17 @@ If you copy to a different folder, update the `backup_dir` variable in the `/etc
 You can either mount the shared storage or copy the backup files to the `/backup/` folder on the target server.
 . Power off the source server.
 . Enter the following commands to register to the Customer Portal, attach subscriptions, and enable only the required subscriptions:
+** If the target server runs {EL}{nbsp}9:
++
+[options="nowrap" subs="quotes,attributes"]
+----
+# subscription-manager register _your_customer_portal_credentials_
```

Comment:
Command code blocks should contain only one command. I recommend breaking this up into separate commands.

Reply:
I'm adding the commands in one block because that's how they are documented for EL 8 (see the command block right below these new lines). I'm not too happy about this either but I also very much want to avoid scope creep with this PR. So no change.

Reply:
OK. I'll create a separate bug.